modules/Story/index.ts

```diff
@@ -1 +1,2 @@
-export { default } from './Story';
+// default export works better with Next.js `dynamic()` component import
+export { Story } from './Story';
```

Comment:
This isn't a _default_ export though. The component is refactored to use named export instead, so maybe this comment is outdated?

Reply:
Good point. Will remove it :+1: 

Thanks!

Reply:
e06f8a96490901da6ae3cca3220cc0cbad04c5d2